torch/nn/modules/container.py

```diff
@@ -538,6 +544,11 @@ def __init__(self, parameters: Optional[Mapping[str, 'Parameter']] = None) -> No
         if parameters is not None:
             self.update(parameters)
 
+    def __setstate__(self, state):
+        state['_initialized'] = False
```

Comment:
nit: I don't think this line is actually needed as we use `getattr` with a default value of `False`.

Reply:
It is needed whenever a ParameterList/Dict is loaded which had its `_initialized` attribute set to `True`. This line changes it back to `False` as desired. (For those modules not having the `_initialized` attribute, this line is indeed not necessary.)

Reply:
Ho right. Sorry. Looks good.
This whole thing is too complex for what it does... Too bad we don't have a better way to do this warning.